examples/mixed_signal/tests/Makefile

```
@@ -0,0 +1,85 @@
+# Override this variable to use a VHDL toplevel instead of (System)Verilog
+TOPLEVEL_LANG ?= verilog
+
+TOPLEVEL ?= tb_rescap
+MODULE ?= test_rescap,test_rescap_minimalist
+
+#TOPLEVEL ?= tb_regulator
+#MODULE ?= test_regulator_plot,test_regulator_trim
```

Hmmmmm... this makes me think that rescap and regulator should probably be different examples. But there is a lot of shared code between the two. Thoughts?

Since the examples ship with the distributable, and since we don't have matplotlib as a dependency, I think the full tests shouldn't be run by default.

This example will eventually be a part of the regression, unless specifically discounted. Is that going to be an issue?

---

I had them as separate examples initially but found myself having to tweak the shared code twice, so merged them.
In principle, the analog probe module could move into a cocotb extension with the examples depending on it, but that feels like too much overhead for little gain.

---

I understand. It's not a big deal for an example. It would be nice if you only had to modify one variable, perhaps `MODULE` can be derived from `TOPLEVEL`?

```suggestion
TOPLEVEL ?= tb_rescap

ifeq ($(TOPLEVEL),tb_rescap)
     MODULE ?= test_rescap,test_rescap_minimalist
else ifeq ($(TOPLEVEL),tb_regulator)
     MODULE ?= test_regulator_plot,test_regulator_trim
else
    $(error Given TOPLEVEL not supported)
endif
```

---

Thanks, that's a good idea.